### PR TITLE
Suggestions enhancement

### PR DESCRIPTION
**Pull request by SavvyShah**
- Document suggestions story

### CODE REVIEW

File: stories/index.js [hunk 1]
```diff
@@ -1929,6 +1930,165 @@ storiesOf("Search components/SearchBox", module)
       <SearchBoxRSDefault placeholder="Search Books..." />
     )
   )
+  .add(
```
Comment (mohdashraf010897) [inline stories/index.js]: @SavvyShah duplicate story

File: stories/index.js
Comment (mohdashraf010897) [inline stories/index.js:1]: Please check the additions, the PR should just add the relevant stories for document suggestions just after the 'faqsuggestions'(or which ever was added last) related stories, the intention is to maintain chronological order.

File: stories/reactivesearch/SearchBox.stories.js [hunk 2]
```diff
@@ -20,6 +20,7 @@ export default class SearchBoxDefault extends Component {
 						<SearchBox
 							dataField={[{ field: "original_title", weight: 3 }, { field: "original_title.search", weight: 1 }]}
 							componentId="BookSensor"
+							showDistinctSuggestions
```
Comment (mohdashraf010897) [inline stories/reactivesearch/SearchBox.stories.js:22]: @SavvyShah  is this required by default to be `true`?

File: stories/reactivesearch/SearchBoxWithDocumentSuggestions.stories.js
Comment (mohdashraf010897) [inline stories/reactivesearch/SearchBoxWithDocumentSuggestions.stories.js:1]: @SavvyShah Can you explain the need for having a separate file for DocumentSuggestions? I would say we utilize the existing [SearchBoxDefault](https://github.com/appbaseio/playground/blob/feat/suggestions-enhancement/stories/reactivesearch/SearchBox.stories.js) file.

File: stories/reactivesearch/movie-store.css
Comment (mohdashraf010897) [inline stories/reactivesearch/movie-store.css:1]: @SavvyShah Please help me understand the use of this css file.

**Review by mohdashraf010897 — CHANGES_REQUESTED** [review]
Please see the review comments and the loom below

https://www.loom.com/share/6f9805bf18a84ea49b02f7171b48c21a?sid=54bca7ec-dfec-44d6-8db0-42ae0d3ff919

File: stories/reactivesearch/SearchBoxWithDocumentSuggestions.stories.js
Comment (SavvyShah) [reply stories/reactivesearch/SearchBoxWithDocumentSuggestions.stories.js:1]: The `good-books-ds` index didn't have document suggestions. That's why I made a new file.

File: stories/index.js (hunk 1 above)
Comment (SavvyShah) [reply stories/index.js]: OK

File: stories/reactivesearch/SearchBox.stories.js (hunk 2 above)
Comment (SavvyShah) [reply stories/reactivesearch/SearchBox.stories.js:22]: Yes. I'll remove it then.

File: stories/reactivesearch/movie-store.css
Comment (SavvyShah) [reply stories/reactivesearch/movie-store.css:1]: It's styling for the document suggestions file. It can be reused when we are using `movie-store` index.

**Pull request merged**